cmd/docker.go

```diff
@@ -48,7 +51,7 @@ func (d *KoolDocker) Execute(args []string) (err error) {
 	image := args[0]
 	workDir, _ := os.Getwd()
 
-	if !d.Flags.DisableTty && !environment.IsTrue("KOOL_TTY_DISABLE") {
+	if !d.Flags.DisableTty && !environment.IsTrue("KOOL_TTY_DISABLE") && d.terminal.IsTerminal(d.GetWriter()) {
```

Comment:
so, with this change we want to remove (deprecate) the flag and the environment variable, so they are not needed anymore.

Reply:
I guess we can do that, and target this to `2.0.0`.

Reply:
@fabriciojs just to confirm: deprecate or remove them?

Reply:
Remove!

Reply:
well, actually, what do you think? I would say remove, but that requires us to go to `2.0.0`... if we just deprecate, we keep the flag - change its documentation to DEPRECATED with NO BEHAVIOUR attached to it, and then we can keep version `1.3.x`... maybe that is better short term?

Reply:
@fabriciojs I have no idea if anyone is using these flags. But when in doubt, I choose the safest way: deprecating.

Reply:
Let's do with deprecation, and target removal at 2.0 👍🏻

Reply:
@fabriciojs Nice! Warn about deprecated stuff on documentation is enough, or should we output a message when deprecated features are used as well?

Reply:
I am good with only documentation, which we already get by updating the flag description.